modules/src/ics03_connection/handler/conn_open_try.rs

```diff
@@ -5,7 +5,7 @@ use crate::ics03_connection::error::{Error, Kind};
 use crate::ics03_connection::handler::verify::{check_client_consensus_height, verify_proofs};
 use crate::ics03_connection::handler::ConnectionEvent::ConnOpenTry;
 use crate::ics03_connection::handler::ConnectionResult;
-use crate::ics03_connection::msgs::MsgConnectionOpenTry;
+use crate::ics03_connection::msgs::conn_open_try::MsgConnectionOpenTry;
```

Comment:
In a separate, later PR, we can consider using the trick Greg used in ibc-proto, to shorten the import paths. So instead of

```
use crate::ics03_connection::msgs::conn_open_try::MsgConnectionOpenTry;
```

we could get 
```
use crate::ics03_connection::msgs::MsgConnectionOpenTry;
```

by using in msgs.rs something like:
```
mod conn_open_try;
pub use conn_open_try::*;
```

This improvement could benefit multiple other parts of our codebase, not just msgs. I will open an issue for this later on.

Reply:
Agreed. I tried this briefly but couldn't make it work and ran out of time.

Reply:
Ah in fact there is already an issue open for this: https://github.com/informalsystems/ibc-rs/issues/186. We'll solve this later!